### PR TITLE
Bug fix/ mi7] rename method to match regular expression

### PR DESCRIPTION
**Pull request by yunaa0414**
**What's in the PR**
* The method name of the function should conform to the naming convention of the function - regular expression

**How to test manually**
* ...

**Automatic testing**
* [ ] PR includes unit tests

**Documentation**
* [ ] PR updates documentation

### CODE REVIEW

**Review by yunaa0414 — COMMENTED** [review]
The method name of the function should conform to the naming convention of the function - regular expression

File: inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/trie/Trie.java [hunk 1]
```diff
@@ -210,7 +210,7 @@ public Node getNode(final CharSequence key)
      *            the length to match.
      * @return the node.
      */
-    private Node _getNode(final CharSequence key, final int offset, final int length)
+    private Node getNode1(final CharSequence key, final int offset, final int length)
```
Comment (munterkalmsteiner) [inline inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/trie/Trie.java:213]: But why adding a suffix of "1" to the method name?

File: inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/querybuilder/SPARQLQueryBuilder.java [hunk 2]
```diff
@@ -663,19 +663,19 @@ public SPARQLQueryBuilder withLabelMatchingExactlyAnyOf(String... aValues)
         IRI ftsMode = forceDisableFTS ? FTS_NONE : kb.getFullTextSearchIri();
         
         if (FTS_LUCENE.equals(ftsMode)) {
-            addPattern(PRIMARY, withLabelMatchingExactlyAnyOf_RDF4J_FTS(aValues));
```
Comment (munterkalmsteiner) [inline inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/querybuilder/SPARQLQueryBuilder.java:666]: Hm, all these changes where the underscore is removed, I'm not sure if this is actually an improvement in readability. The underscore somehow chunked the long variable names so they are easier to read, in my opinion.

**Review by munterkalmsteiner — APPROVED** [review]
I approve the refactoring, even though I don't agree that it improves code readability.

**Pull request closed without merging**